### PR TITLE
Update to drand 0.8.1

### LINKED ISSUE #4048: Drand Upgrade
### Description
Biggest change is that we will never skip a round which simplifies things drastically.
Other change is that we no longer get previous round from response

### PR DESCRIPTION
**Pull request by anorth**
### Motivation
DRAND protocol changed to remove the parent round and have no gaps.

This is the minimum change to consume the new protocol. The changes mean we now have a bunch of unnecessary catch-up logic; clean-up is deferred to #4053

### Proposed changes
Remove use of the DRAND previous round number. Assume rounds are contiguous.

Closes #4048

### CODE REVIEW

File: internal/pkg/block/block.go [hunk 1]
```diff
@@ -33,9 +33,9 @@ type Block struct {
 	// ElectionProof is the vrf proof giving this block's miner authoring rights
 	ElectionProof *crypto.ElectionProof
 
-	// DrandEntries contain the verifiable oracle randomness used to elect
+	// BeaconEntries contain the verifiable oracle randomness used to elect
 	// this block's author leader
-	DrandEntries []*drand.Entry
```
Comment (anorth) [inline internal/pkg/block/block.go:38]: I'd prefer the nomenclature less tightly coupled to DRAND throughout. I played with renaming the `drand` package to `beacon` but it was too much noise in this change.

Comment (ZenGround0) [reply internal/pkg/block/block.go:38]: Yes this is a good idea. I'll file an issue

File: internal/pkg/drand/drand_grpc.go [hunk 2]
```diff
@@ -213,6 +212,8 @@ func (d *GRPC) StartTimeOfRound(round Round) time.Time {
 // 1. RoundsInInterval is called on an interval extending into the future
 // 2. There is an ongoing drand outage causing a gap in drand rounds
 // 3. There is a network partition blocking this GPRC from the latest rounds
+// TODO remove gap-handling logic now that the protocol has no gaps
+// https://github.com/filecoin-project/go-filecoin/issues/4053
```
Comment (ZenGround0) [inline internal/pkg/drand/drand_grpc.go:216]: 🎉

Comment (ZenGround0) [reply internal/pkg/drand/drand_grpc.go:216]: #4055

**Pull request merged**